test/e2e/antreapolicy_test.go

```diff
@@ -3000,6 +3001,246 @@ func testACNPICMPSupport(t *testing.T, data *TestData) {
+	testNamespace := data.testNamespace
+	senderName, _, cleanupFunc := createAndWaitForPod(t, data, data.createMcJoinPodOnNode, "test-sender-", nodeName(mc.senderConfig.nodeIdx), testNamespace, mc.senderConfig.isHostNetwork)
+	defer cleanupFunc()
+	var wg sync.WaitGroup
+	receiverNames, cleanupFuncs := setupReceivers(t, data, mc, mcjoinWaitTimeout, &wg)
+	for _, cleanupFunc := range cleanupFuncs {
+		defer cleanupFunc()
+	}
+	// Wait 2 seconds(-w 2) before sending multicast traffic.
+	// It sends two multicast packets for every second(-f 500 means it takes 500 milliseconds for sending one packet).
+	sendMulticastCommand := []string{"/bin/sh", "-c", fmt.Sprintf("mcjoin -f 500 -o -p %d -s -t 3 -w 2 -W %d %s", mc.port, mcjoinWaitTimeout, mc.group.String())}
+	go func() {
+		data.RunCommandFromPod(testNamespace, senderName, mcjoinContainerName, sendMulticastCommand)
+	}()
+
+	tcpdumpName, _, cleanupFunc := createAndWaitForPod(t, data, data.createNetshootPodOnNode, "test-tcpdump-", nodeName(mc.receiverConfigs[0].nodeIdx), testNamespace, true)
+	defer cleanupFunc()
+
+	groupAddress := "224.0.0.1"
+	cmd, err := generatePacketCaptureCmd(data, 15, groupAddress, nodeName(mc.receiverConfigs[0].nodeIdx), receiverNames[0])
+	if err != nil {
+		t.Fatalf("failed to call generateConnCheckCmd: %v", err)
+	}
+
+	// check if IGMP can be sent to Pod
+	if err := wait.Poll(3*time.Second, defaultTimeout, func() (bool, error) {
+		captured, _, _, err := checkPacketCaptureResult(data, tcpdumpName, cmd)
+		if captured {
+			return true, nil
+		}
+		return false, err
+	}); err != nil {
+		t.Fatalf("Error waiting for multicast routes and stats: %v", err)
+	}
+	t.Logf("waitting for multicast receivers to be ready")
+	wg.Wait()
+	label := "igmp-query"
+	acnpName := name
+	_, err = k8sUtils.LabelPod(testNamespace, receiverNames[0], "antrea-e2e", label)
+	if err != nil {
+		t.Fatalf("failed to label pod %s: err=%v", receiverNames[0], err)
+	}
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName(acnpName).SetPriority(1.0).
+		SetAppliedToGroup([]ACNPAppliedToSpec{{PodSelector: map[string]string{"antrea-e2e": label}}})
+
+	// create acnp with ingress rule fir IGMP query
+	igmpType := crdv1alpha1.IGMPQuery
+	builder.AddIngress(ProtocolIGMP, nil, nil, nil, nil, nil, &igmpType, &groupAddress, nil, nil, nil,
+		nil, nil, false, nil, action, "", "", nil)
+	acnp := builder.Get()
+	_, err = k8sUtils.CreateOrUpdateACNP(acnp)
+	defer data.crdClient.CrdV1alpha1().ClusterNetworkPolicies().Delete(context.TODO(), acnp.Name, metav1.DeleteOptions{})
+	if err != nil {
+		t.Fatalf("failed to create acnp %v: %v", acnpName, err)
+	}
+
+	// check if IGMP is dropped or not based on rule action
+	captured, stdout, stderr, err := checkPacketCaptureResult(data, tcpdumpName, cmd)
+	t.Logf("%s returned: stdout %v, std err : %v", cmd, stdout, stderr)
+	if action == crdv1alpha1.RuleActionAllow {
+		if !captured || err != nil || strings.Contains(stderr, "0 packets captured") {
```

Comment:
` strings.Contains(stderr, "0 packets captured") ` is checked in function `checkPacketCaptureResult`

Reply:
done